code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -634,7 +636,19 @@ This is here to make the tiles around the station mininuke change when it's arme
 		STOP_PROCESSING(SSobj, src)
 		CRASH("A fake nuke disk tried to call process(). Who the fuck and how the fuck")
 	var/turf/newturf = get_turf(src)
+
 	if(newturf && lastlocation == newturf)
+
+	// How comfy is disky?
+		var/disk_comfort_level = 0
+
+		// Checking for items that make disky comfy
+		for(var/obj/comfort_item in loc)
+			if(istype(comfort_item, /obj/item/bedsheet) || istype(comfort_item, /obj/structure/bed))
+				disk_comfort_level++
+
+		if(disk_comfort_level >= 2) //Sleep tight, disky.
+			visible_message("<span class='notice'>[src] sleeps soundly. Sleep tight, disky.</span>")
```

Comment:
this doesn't prevent the loneop stuff from triggering, right?

Reply:
Huh, looks like it. Guess I was just hallucinarong it having the return in there.

Reply:
actually won't this end up triggering every single process() if it's stationary on a bed..?